index.html

```diff
@@ -38,9 +39,137 @@ <h2 class="intro profession">I'am a Software Developer</h2>
             <li><a href="#" class="fa fa-medium"></a></li>
         </ul>
     </div>
+</section>
+
+<section class="item-2">
+    <div class="recentWork">
+        <h1>My Recent Works</h1>
+    </div>
+    <img class="line" src="assets/line.png" alt="straight line">
+    <div class="works first">
+        <img class="firstProjectImg" src="./assets/imgplaceholder.png" alt="project Image">
+        <h2 class="multi"> Multi-Post Stories</h2>
+        <p class="story">A daily selection of privately personalized reads; no accounts or sign-ups required. has been the industry's standard dummy text ever since the 1500s, when an unknown printer took a standard dummy text.</p>
+        <div class="grid-container">
```

Comment:
- [ ] Use a `ul` tag instead of `div` and use `li` tag to list the technologies.